run.py

```diff
@@ -0,0 +1,492 @@
+#!/usr/bin/env python3
+
+import base64
+import copy
+import csv
+import datetime
+import flywheel
+import json
+import logging
+import os
+import pprint
+import re
+import shutil
+import sys
+import tempfile
+import zipfile
+from urllib.parse import urljoin
+
+from flywheel_migration.dcm import DicomFile
+from flywheel_migration.util import DEFAULT_TZ
+from healthcare_api.client import Client as HealthcareAPIClient
+
+import dateutil.parser
+import pytz
+import requests
+from dicomweb_client.api import load_json_dataset
+from requests_toolbelt.multipart.encoder import MultipartEncoder
+
+log = logging.getLogger('ghc_import')
+
+
+HL7_SEX_MAPPING = {
+    'F': 'female',
+    'M': 'male',
+    'O': 'other',
+    'U': 'unknown'
+}
+
+HL7_ETHNIC_GROUP_MAP = {
+    'H': 'Hispanic or Latino',
+    'N': 'Not Hispanic or Latino',
+    'U': 'Unknown or Not Reported',
+}
+
+
+def main(context):
+    config = context.config
+
+    log.setLevel(getattr(logging, config['log_level']))
+
+    api_key = context.get_input('key')['key']
+
+    if 'docker.local.flywheel.io' in api_key:
+        # dev workaround for accessing docker.local - set own host ip
+        # ip -o route get to 8.8.8.8 | sed -n 's/.*src \([0-9.]\+\).*/\1/p'
+        host_ip = '192.168.50.162'
+        with open('/etc/hosts', 'a') as f:
+            f.write(host_ip + '\tdocker.local.flywheel.io\n')
+    api_uri = api_key.rsplit(':', 1)[0]
+    if not api_uri.startswith('http'):
+        api_uri = 'https://' + api_uri + '/api'
+    fw_api = FwApi(api_uri, api_key)
+
+    # validate destination project exists
+    resp = fw_api.get('projects/' + config['project_id'])
```

Comment:
I think we should minimize the usage of custom api clients. I suppose that we can use the SDK to validate that the project exists or not.
@ambrussimon any thoughts?

Reply:
Never mind, I will open a PR soon which includes this change.